src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArray = century
+    ? [...people].filter(person => person.sex === 'm'
```

Comment:
filter return new array
fix in all cases
```suggestion
    ? people.filter(person => person.sex === 'm'
```